spotifycli/spotifycli.py

```diff
@@ -118,27 +118,28 @@ def add_arguments():
 
 def get_arguments():
     return [
-        ("--version", "shows version number"),
-        ("--status", "shows song name and artist"),
+        ("--version", "shows version number", False),
+        ("--status", "shows song name and artist", False),
         (
             "--statusposition",
-            "shows song name and artist, with current playback position"
+            "shows song name and artist, with current playback position",
+            False
         ),
-        ("--statusshort", "shows status in a short way"),
-        ("--song", "shows the song name"),
-        ("--songshort", "shows the song name in a short way"),
-        ("--artist", "shows artist name"),
-        ("--artistshort", "shows artist name in a short way"),
-        ("--album", "shows album name"),
-        ("--position", "shows song position"),
-        ("--arturl", "shows album image url"),
-        ("--playbackstatus", "shows playback status"),
-        ("--play", "plays the song"),
-        ("--pause", "pauses the song"),
-        ("--playpause", "plays or pauses the song (toggles a state)"),
-        ("--lyrics", "shows the lyrics for the song"),
-        ("--next", "plays the next song"),
-        ("--prev", "plays the previous song"),
+        ("--statusshort", "shows status in a short way", False),
+        ("--song", "shows the song name", False),
+        ("--songshort", "shows the song name in a short way", False),
+        ("--artist", "shows artist name", False),
+        ("--artistshort", "shows artist name in a short way", False),
+        ("--album", "shows album name", False),
+        ("--position", "shows song position", False),
+        ("--arturl", "shows album image url", False),
+        ("--playbackstatus", "shows playback status", False),
+        ("--play", "plays the song", False),
+        ("--pause", "pauses the song", False),
+        ("--playpause", "plays or pauses the song (toggles a state)", False),
+        ("--lyrics", "shows the lyrics for the song", False),
+        ("--next", "plays the next song", False),
+        ("--prev", "plays the previous song", False),
```

Comment:
I _think_ these are still needed, right?  I can't quite tell where they went.  But if I'm wrong and you removed them intentionally, let me know and I'll be happy to update it!